4-projects/modules/ml_infra_projects/artifacts_project.tf

```diff
@@ -15,6 +15,7 @@
  */
 
 locals {
+  artifacts_project_suffix = "artifacts"
```

Comment:
If this is just a hard-coded string, wouldn't be better to have a `variable` instead a `local`?

Reply:
good idea, will implement this

Reply:
I don't know if that would be the correct logic. But I would use a hardcoded value in `locals` for a default value that does not need or should not be modified by the user. I would use a `variable` if a value that can be defined/changed by the user is needed.